### ISSUE TITLE
The screen flicker when writing in present/speaker mode

### ISSUE THREAD

**Issue by wcshds** [typst/webapp-issues#409]
### Description

I'm trying to use the present/speaker mode in the Typst web app, but why does the screen flicker when I write on the slide? (I'm using a graphics tablet)

https://github.com/user-attachments/assets/c36116e5-a720-48b2-b6ae-dad20ea42737

### Browsers

Chrome

### OS

Windows

**Comment by laurmaedje** [comment]
That's very strange. Does it only happen when using the tablet?

**Comment by wcshds** [comment]
> That's very strange. Does it only happen when using the tablet?

Yes, it only happens when using the tablet. I'm using the wacom pth-660.
